Src/IntegrationTests/Tests/Cache/GetTests.cs

```diff
@@ -0,0 +1,36 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace Tests.Cache
+{
+    public class GetTests : IClassFixture<WebApplicationFactory<TestMvcApp.Startup>>
+    {
+        private readonly WebApplicationFactory<TestMvcApp.Startup> _factory;
+
+        public GetTests(WebApplicationFactory<TestMvcApp.Startup> factory)
+        {
+            this._factory = factory;
+        }
+
+        [Fact]
+        public async Task Get_Cache_Working()
+        {
+            var client = _factory.CreateClient();
+
+            var response = await client.GetStringAsync("/Cache/Get");
```

Comment:
I would make the endpoint **only** use the `Get` method. So for the test you would do:

```
await client.GetStringAsync("/Cache/Remember");
var response = await client.GetStringAsync("/Cache/Get");
```

This would verify that the cache is working across HTTP requests 👍